internal/registry/git_test.go

```diff
@@ -0,0 +1,78 @@
+package registry_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/heroku/color"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	ilogging "github.com/buildpacks/pack/internal/logging"
+	"github.com/buildpacks/pack/internal/registry"
+	"github.com/buildpacks/pack/logging"
+	h "github.com/buildpacks/pack/testhelpers"
+)
+
+func TestGit(t *testing.T) {
+	color.Disable(true)
```

Comment:
nit: If you're not testing any logging, disabling and reenabling color isn't needed here